:memo: Document the fields of `values.yaml` in Helm chart's `README.md` (#1437)
kubeshark/kubeshark: helm-chart/README.md

helm install kubeshark kubeshark/kubeshark \
  --set tap.ipv6=false
```

## Configuration

| Parameter                                 | Description                                   | Default                                                 |
|-------------------------------------------|-----------------------------------------------|---------------------------------------------------------|

> | `tap.docker.registry`                     | Docker registry URL                           | `docker.io/kubeshark`                                   |

Reviewer: ```suggestion
| `tap.docker.registry`                     | Docker registry to pull from                           | `docker.io/kubeshark`                                   |
```

| `tap.docker.tag`                          | Docker image tag                              | `latest`                                                |
| `tap.docker.imagePullPolicy`              | Kubernetes image pull policy                  | `Always`                                                |

> | `tap.docker.imagePullSecrets`             | Kubernetes secrets to pull Docker images      | `[]`                                                    |

Reviewer: ```suggestion
| `tap.docker.imagePullSecrets`             | Kubernetes secrets to pull the images      | `[]`                                                    |
```

| `tap.proxy.worker.srvport`                | Worker service port                           | `8897`                                                  |
| `tap.proxy.hub.port`                      | Hub service port                              | `8898`                                                  |

> | `tap.proxy.hub.srvport`                   | Hub service port (internal)                   | `8898`                                                  |

Reviewer: ```suggestion
| `tap.proxy.hub.srvport`                   | Hub server port                   | `8898`                                                  |
```

| `tap.proxy.front.port`                    | Front-facing service port                     | `8899`                                                  |

> | `tap.proxy.host`                          | Allowed IPs                                   | `127.0.0.1`                                             |

Reviewer: ```suggestion
| `tap.proxy.host`                          | Proxy server's IP                                   | `127.0.0.1`                                             |
```

> | `tap.namespaces`                          | List of namespaces to observe                 | `[]`                                                    |

Reviewer: ```suggestion
| `tap.namespaces`                          | List of namespaces for the traffic capture                 | `[]`                                                    |
```

> | `tap.release.repo`                        | URL for the Helm chart repository             | `https://helm.kubeshark.co`                             |

Reviewer: ```suggestion
| `tap.release.repo`                        | URL of the Helm chart repository             | `https://helm.kubeshark.co`                             |
```

| `tap.release.name`                        | Name for the release                          | `kubeshark`                                             |

> | `tap.release.namespace`                   | Namespace to install the chart                | `default`                                               |

Reviewer: ```suggestion
| `tap.release.namespace`                   | Helm release namespace                | `default`                                               |
```

> | `tap.persistentstorage`                   | Enable using persistent storage for the worker | `false`                                                |

Reviewer: ```suggestion
| `tap.persistentstorage`                   | Use `persistentVolumeClaim` instead of `emptyDir` | `false`                                                |
```

> | `tap.storagelimit`                        | Limit for persistent storage                  | `200Mi`                                                 |

Reviewer: ```suggestion
| `tap.storagelimit`                        | Limit of either the `emptyDir` or `persistentVolumeClaim`                  | `500Mi`                                                 |
```

| `tap.storageclass`                        | Storage class for persistent storage          | `standard`                                              |

> | `tap.dryrun`                              | Opt-out of collecting data                    | `false`                                                 |

Reviewer: ```suggestion
| `tap.dryrun`                              | Preview of all pods matching the regex, without tapping them                    | `false`                                                 |
```

| `tap.resources.worker.requests.cpu`       | CPU request for worker                        | `50m`                                                   |
| `tap.resources.worker.requests.memory`    | Memory request for worker                     | `50Mi`                                                  |
| `tap.resources.hub.limits.cpu`            | CPU limit for hub                             | `750m`                                                  |
| `tap.resources.hub.limits.memory`         | Memory limit for hub                          | `1Gi`                                                   |
| `tap.resources.hub.requests.cpu`          | CPU request for hub                           | `50m`                                                   |
| `tap.resources.hub.requests.memory`       | Memory request for hub                        | `50Mi`                                                  |
| `tap.servicemesh`                         | Enable integration with service mesh          | `true`                                                  |

> | `tap.tls`                                 | Listen to TLS traffic                         | `true`                                                  |

Reviewer: ```suggestion
| `tap.tls`                                 | Capture the encrypted/TLS traffic from cryptography libraries like OpenSSL                         | `true`                                                  |
```

> | `tap.packetcapture`                       | Packet capture library to use (options available?) | `libpcap`                                          |

Reviewer: ```suggestion
```

| `tap.ignoretainted`                       | Whether to ignore tainted nodes               | `false`                                                 |
| `tap.labels`                              | Kubernetes labels to apply to Tap components  | `{}`                                                    |
| `tap.annotations`                         | Kubernetes annotations to apply to Tap components | `{}`                                                |
| `tap.nodeselectorterms`                   | Node selector terms                           | `[{"matchExpressions":[{"key":"kubernetes.io/os","operator":"In","values":["linux"]}]}]` |
| `tap.auth.enabled`                        | Enable authentication                         | `false`                                                 |

> | `tap.auth.approvedemails`                 | List of approved email addresses              | `[]`                                                    |

Reviewer: ```suggestion
| `tap.auth.approvedemails`                 | List of approved email addresses for authentication              | `[]`                                                    |
```

> | `tap.auth.approveddomains`                | List of approved email domains                | `[]`                                                    |

Reviewer: ```suggestion
| `tap.auth.approveddomains`                | List of approved email domains for authentication                | `[]`                                                    |
```

> | `tap.ingress.enabled`                     | Enable ingress                                | `false`                                                 |

Reviewer: ```suggestion
| `tap.ingress.enabled`                     | Enable `Ingress`                                | `false`                                                 |
```

| `tap.ingress.classname`                   | Ingress class name                            | `""`                                                    |

> | `tap.ingress.host`                        | Host for the ingress                          | `ks.svc.cluster.local`                                  |

Reviewer: ```suggestion
| `tap.ingress.host`                        | Host of the `Ingress`                          | `ks.svc.cluster.local`                                  |
```

> | `tap.ingress.tls`                         | Ingress TLS configuration                     | `[]`                                                    |

Reviewer: ```suggestion
| `tap.ingress.tls`                         | `Ingress` TLS configuration                     | `[]`                                                    |
```

> | `tap.ingress.annotations`                 | Ingress annotations                           | `{}`                                                    |

Reviewer: ```suggestion
| `tap.ingress.annotations`                 | `Ingress` annotations                           | `{}`                                                    |
```

| `tap.ipv6`                                | Allow support for IPv6                        | `true`                                                  |
| `tap.debug`                               | Enable debug mode                             | `false`                                                 |
| `tap.nokernelmodule`                      | Use TAP's eBPF instead of kernel module       | `false`                                                 |

> | `tap.telemetry.enabled`                   | Enable anonymous telemetry tracking           | `true`                                                  |

Reviewer: ```suggestion
| `tap.telemetry.enabled`                   | Enable anonymous usage statistics collection           | `true`                                                  |
```

> | `logs.file`                               | Output file for the logs                      | `""`                                                    |

Reviewer: ```suggestion
| `logs.file`                               | Logs dump path                      | `""`                                                    |
```

> | `kube.configpath`                         | Path to the Kubernetes config file            | `""`                                                    |

Reviewer: ```suggestion
| `kube.configpath`                         | Path to the `kubeconfig` file (`$HOME/.kube/config`)            | `""`                                                    |
```

| `kube.context`                            | Kubernetes context to use for the deployment  | `""`                                                    |
| `dumplogs`                                | Enable dumping of logs to the console         | `false`                                                 |
| `headless`                                | Enable running in headless mode               | `false`                                                 |
| `license`                                 | License key for the Pro/Enterprise edition    | `""`                                                    |

> | `scripting.env`                           | Environment variables for scripted rules      | `{}`                                                    |

Reviewer: ```suggestion
| `scripting.env`                           | Environment variables for the scripting      | `{}`                                                    |
```

> | `scripting.source`                        | Source file for scripted rules                | `""`                                                    |

Reviewer: ```suggestion
| `scripting.source`                        | Source directory of the scripts                | `""`                                                    |
```

> | `scripting.watchscripts`                  | Enable watch mode for scripted rules          | `true`                                                  |

Reviewer: ```suggestion
| `scripting.watchscripts`                  | Enable watch mode for the scripts in source directory          | `true`                                                  |
```
